sessionctx/variable/sysvar.go

```diff
@@ -730,6 +730,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, TiDBEnableStmtSummary, "0"},
 	{ScopeGlobal | ScopeSession, TiDBCapturePlanBaseline, "0"},
 	{ScopeGlobal | ScopeSession, TiDBUsePlanBaselines, BoolToIntStr(DefTiDBUsePlanBaselines)},
+	{ScopeGlobal | ScopeSession, TiDBIsolationReadEngines, ""},
```

Comment:
the default value should be `TiKV`?

Reply:
I prefer `TiKV,TiFlash`. What your opinion?